metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1597,17 +1604,17 @@ class MetalsLanguageServer(
       for {
         _ <- disconnectOldBuildServer()
         maybeBuild <- timed("connected to build server") {
-          if (buildTools.isBloop) bloopServers.newServer(userConfig)
-          else bspServers.newServer()
+          bspConnector.connect(workspace, userConfig, buildTools)
         }
         result <- maybeBuild match {
           case Some(build) =>
             val result = connectToNewBuildServer(build)
-            build.onReconnection { reconnected =>
-              connectToNewBuildServer(reconnected)
-                .flatMap(compileAllOpenFiles)
-                .ignoreValue
-            }
+            // TODO how to implement reconnection correctly ?
```

Comment:
I'm not sure how to implement this thing correctly for multiply connections.

When reconnect happens?
I've tried to kill `bloop.Server` but this part of code wasn't executed

Reply:
I'm not quite sure about this change.

https://github.com/scalameta/metals/blob/95e8daf51964c373b769f07788e2edd1570889ee/metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala#L1630-L1636

Would it be ok to leave it in this way?